core/src/main/java/io/parsingdata/metal/token/Token.java

```diff
@@ -16,24 +16,24 @@
 
 package io.parsingdata.metal.token;
 
-import java.io.IOException;
-
 import io.parsingdata.metal.data.Environment;
 import io.parsingdata.metal.data.ParseResult;
 import io.parsingdata.metal.encoding.Encoding;
 
+import java.io.IOException;
+
 public abstract class Token {
 
     public final static String DEFAULT_NAME = "W";
 
-    private final Encoding _enc;
+    public final Encoding enc;
```

Comment:
Create a `enc` -> `encoding` and `env` -> `environment` issue for the 5.0.0 milestone.